contracts/PushCore/PushCoreV3.sol

```diff
@@ -296,27 +295,38 @@ contract PushCoreV3 is
         // Check channel's current state
```

Comment:
Is there any reason to still have the updateChannelState function and then it routes to specific ones
instead of making deactivate and reactivate external itself
is it more bytecode efficient?

Reply:
1. The reason to have a separate external function for reactivating is that it transfers tokens from the sender to the core. 

2. For `deactivateChannel`, even though it doesn't transfer any token from `msg.sender`, a separate function is important because the caller needs to be the channel Owner. We can't expect the same behavior for both external and CCR usage using a common function.

Reply:
No, I meant why don't we have 'reactivate' and 'deactivate' as standalone external functions? 
Currently, 'updateChannelState' is the main function users must invoke to use these features.
@Zartaj0